.travis.yml

```diff
@@ -13,8 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-sudo: false
 before_install: git fetch --unshallow
 language: java
-install: mvn install --batch-mode -DskipTests=true -Dmaven.javadoc.skip=true -Dsource.skip=true > mvn_install.log || (cat mvn_install.log && false)
-script: mvn package -DskipTests=true
+jdk:
+  - openjdk7
```

Comment:
@ilooner-mapr Switch to jdk7 is not required, it works with jdk8 as well, but enforcing jdk7 helps with validating that the project is java 1.7 compliant, so I am OK with the change.

Reply:
Hmm good to know.